examples/qualcomm/oss_scripts/llama3_2/runner/runner.cpp

```diff
@@ -43,12 +43,12 @@ Runner::Runner(
     const std::vector<std::string>& models_path,
     const std::string& tokenizer_path,
     const float temperature)
-    : tokenizer_path_(tokenizer_path),
-      temperature_(temperature),
-      n_bos_(1),
+    : n_bos_(1),
```

Comment:
@shewu-quic can you take a look and see if the change in the runner makes sense?